AlphaWallet/Settings/Types/Constants.swift

```diff
@@ -226,6 +226,10 @@ public struct Constants {
         static let connectionTimeout: TimeInterval = 10
     }
 
+    enum Ens {
+        static let recordExpiration: TimeInterval = -(7 * 24 * 60 * 60)
```

Comment:
I think this should be positive, instead of negative? Because: `NSDate(timeIntervalSinceNow: expirationTime)`

Reply:
since we want to check if date has expired, we get now and subtract `n` timeinterval to figure out `7 day ago`, and later check, if creation date is greater than our date

Reply:
Oh, Ok.